### DISCUSSION TITLE
getMarkerFeature(): Found less than 100 cells for background matching, Lowering k to 0

### DISCUSSION THREAD

**Discussion by Jiahuan90** [GreenleafLab/ArchR#1463] [Questions / Documentation] [answered]
Hi rcorces,

I have encountered a problem: one of cell type number in ArchR project is 86 cells. When I ran getMarkerFeature() function to descover differential peaks among clusters, it met an error: "Found less than 100 cells for background matching, Lowering k to 0".

If the rare number cell type is of biological significance, how can I set parameters to overcome this problem? 
Below is my code:

    markersPeaks <- getMarkerFeatures(
                                  ArchRProj = proj,
                                  useMatrix = "PeakMatrix",
                                  groupBy = "cell.type",
                                  bias = c("TSSEnrichment", "log10(nFrags)"),
                                  testMethod = "wilcoxon"
  )
     
Thank you for your kindly help~~

**Comment by rcorces** [comment, marked as answer]
86 cells is simply not enough to make biologically meaningful conclusions from. There is really no way around that.

**Reply by Jiahuan90** [reply]
Thank you for your kindly help~
